docs/content/en/docs/data/data-source/scan_sql.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "scan_sql"
+linkTitle: "scan_sql"
+weight: 210
+superheading: "catalog_data_source."
+---
+
+
+
+``scan_sql(data_source_id: str, sql_request: ScanSqlRequest)``
+
+Analyze SELECT SQL query in the request. Return description of SQL result-set as list of column names with GoodData data types and list of example data returned by SQL.
+
+{{% parameters-block  title="Parameters"%}}
+
+{{< parameter p_name="data_source_id" p_type="string" >}}
+Data source identification string. e.g. "demo"
+{{< /parameter >}}
+
+{{< parameter p_name="sql_request" p_type="ScanSqlRequest" >}}
+SELECT SQL query to analyze.
+{{< /parameter >}}
+
+{{% /parameters-block %}}
+
+{{% parameters-block title="Returns"%}}
+
+{{< parameter p_type="ScanSqlResponse" >}}
+Description of SQL result-set as list of column names with GoodData data types and list of example data returned by SQL.
```

Comment:
I would change returned by SQL to returned by the SELECT query.

Reply:
Fixed